pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -20,17 +20,17 @@
 , stdenv
   # Options:
 , cudaSupport ? config.cudaSupport
-, cudaPackagesGoogle
+, cudaPackages
```

Comment:
I don't remember if this was my idea or @SomeoneSerge's but we surely discussed it in private.
It feels like there is no strong reason to keep them synced. Furthermore, our tensorflow version seriously start to lack behind (2.13.0 while latest is 2.16.1).

Reply:
Yeah that's fair. Tensorflow isn't worth slowing us down IMHO

But Shouldn't we get rid of the alias then?

Reply:
I think before jax we also used to have a bunch of `inherit` statements for different variants of tf. If these are still around, we can keep them at `cudaPackagesGoogle`.

- Ideally we'd build everything with the generic `cudaPackages`
- We make an exception for tf because it's so effed up
- If we can avoid making an exception for a package (e.g. jax) we do

Wdyt?

Reply:
> I think before jax we also used to have a bunch of inherit statements for different variants of tf. If these are still around, we can keep them at cudaPackagesGoogle.

AFAICT ([code search](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20cudapackagesgoogle&type=code)), `cudaPackagesGoogle` is only used in jaxlib and TF. In that case, I propose removing the `cudaPackagesGoogle` alias and having TF point to its own desired version of `cudaPackages`. Leaving it around only causes confusion IMHO.

> Wdyt?

Totally agree on all bullet points!

Reply:
Great ! Any modification to do on this PR then ?

Reply:
I propose that we remove the `cudaPackagesGoogle` alias since its ref count is now 1.

Reply:
> alias since its ref count is now 1

It technically isn't and I'm not sure about the proposal. It's still being used to synchronize `tensorflow` and `tensorflow-bin`, and there's no guarantee that jaxlib will always keep up with our default. I would want for the `.override` and `passthru` mess to come back, and having a separate attribute for the slow runners is the easiest way to maintain a fixpoint...

You decide

Reply:
I agree, maybe we should do that in a separate PR (if we do it at all).

Reply:
Ok, fair enough I'm happy to leave it then